dynamic/src/ext.rs

```diff
@@ -1213,6 +1215,17 @@ impl BomaExt for BattleObjectModuleAccessor {
         let next = *((next + 0x8) as *const u64);
         *((next + 0x8) as *const i32)
     }
+
+    unsafe fn is_parry_input(&mut self) -> bool {
+        let is_taunt_buffered = ControlModule::get_trigger_count(self, *CONTROL_PAD_BUTTON_APPEAL_HI as u8) < ControlModule::get_command_life_count_max(self) as i32  // checks if Taunt input was pressed within max tap buffer window
+                                    || ControlModule::get_trigger_count(self, *CONTROL_PAD_BUTTON_APPEAL_S_L as u8) < ControlModule::get_command_life_count_max(self) as i32
+                                    || ControlModule::get_trigger_count(self, *CONTROL_PAD_BUTTON_APPEAL_S_R as u8) < ControlModule::get_command_life_count_max(self) as i32
+                                    || ControlModule::get_trigger_count(self, *CONTROL_PAD_BUTTON_APPEAL_LW as u8) < ControlModule::get_command_life_count_max(self) as i32;
+        let is_special_buffered = ControlModule::get_trigger_count(self, *CONTROL_PAD_BUTTON_SPECIAL as u8) < ControlModule::get_command_life_count_max(self) as i32;  // checks if Special input was pressed within max tap buffer window
+
+        (self.is_button_on(Buttons::TauntParry) && is_taunt_buffered)
+        || (self.is_button_on(Buttons::SpecialParry) && is_special_buffered)
+    }

```

Comment:
... can we ask for `ControlModule::get_command_life_count_max(self) as i32` just once

Reply:
just 4 u